apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/server/WrappedClaimSource.java

```diff
@@ -94,6 +98,14 @@ public static MessageSource<FissClaimChange> wrapFissClaims(
                 .setChangeType(type)
                 .setSeq(seq)
                 .setClaim(claim)
+                .setSource(
+                    RecordSource.newBuilder()
+                        .setPhase("p1")
+                        .setPhaseSeqNum(1)
+                        .setExtractDate(LocalDate.now().minusDays(1).toString())
+                        .setTransmissionTimestamp(
+                            Instant.now().minus(2, ChronoUnit.DAYS).toString())
```

Comment:
All time needs to come from the provided `Clock` so use `clock.instant()` instead of `Instant.now().